pysatModels/models/sami2py_sami2.py

```diff
@@ -0,0 +1,196 @@
+                                "Sami2 is Another Model of the Ionosphere",
+                                "(SAMI2): A new low‐latitude ionosphere",
+                                "model, J. Geophys. Res., 105, Pages",
+                                "23035-23053,",
+                                "https://doi.org/10.1029/2000JA000035,",
+                                "2000.\n",
+                                "Klenzing, J., Jonathon Smith, Michael",
+                                "Hirsch, & Angeline G. Burrell. (2020,",
+                                "July 17). sami2py/sami2py: Version 0.2.2",
+                                "(Version v0.2.2). Zenodo.",
+                                "http://doi.org/10.5281/zenodo.3950564"))
+    logger.info(self.acknowledgements)
+    return
+
+
+# ----------------------------------------------------------------------------
+# Instrument functions
+#
+# Use local and default pysat methods
+
+# Set the list_files routine
+fname = 'sami2py_output_{year:04d}-{month:02d}-{day:02d}.nc'
+supported_tags = {'': {'': fname, 'test': fname}}
+list_files = functools.partial(pysat.instruments.methods.general.list_files,
+                               supported_tags=supported_tags)
+
+
+def load(fnames, tag=None, inst_id=None, **kwargs):
+    """Load sami2py data using xarray.
+
+    This routine is called as needed by pysat. It is not intended
+    for direct user interaction.
+
+    Parameters
+    ----------
+    fnames : array-like
+        iterable of filename strings, full path, to data files to be loaded.
+        This input is nominally provided by pysat itself.
+    tag : str or NoneType
+        tag name used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself. (default=None)
+    inst_id : str or NoneType
+        Instrument ID used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself. (default=None)
+    **kwargs : dict
+        Passthrough for additional keyword arguments specified when
+        instantiating an Instrument object. These additional keywords
+        are passed through to this routine by pysat.
+
+    Returns
+    -------
+    data : xarray.Dataset
+        pysat formatted xarray Dataset
+    meta : pysat.Metadata
+        Model run meta data
+
+    Note
+    ----
+    Any additional keyword arguments passed to pysat.Instrument
+    upon instantiation are passed along to this routine.
+
+    Examples
+    --------
+    ::
+
+        inst = pysat.Instrument('sami2py', 'sami2')
+        inst.load(2019, 1)
+
+    """
+
+    # Load data
+    data, meta = pysat.utils.load_netcdf4(fnames, pandas_format=False)
+
+    # Add time variable for pysat compatibilty
+    data['time'] = [dt.datetime(2019, 1, 1)
+                    + dt.timedelta(seconds=int(val * 3600.0))
+                    for val in data['ut'].values]
+    # Manually close link to file
```

Comment:
This may deserve a note to indicate the close line of code is extra safe.... or a holdover from a pysat bug since quashed, or from issues only on windows, or ....
```suggestion
    # Manually close link to file
```